storage/src/tests/storageserver/bouncertest.cpp

```diff
@@ -283,5 +314,32 @@ BouncerTest::outOfBoundsConfigValuesThrowException()
     } catch (config::InvalidConfigException) {}
 }
 
+
+namespace {
+
+std::shared_ptr<const lib::ClusterStateBundle>
+makeClusterStateBundle(const vespalib::string &baselineState, const std::map<document::BucketSpace, vespalib::string> &derivedStates)
+{
+    lib::ClusterStateBundle::BucketSpaceStateMapping derivedBucketSpaceStates;
+    for (const auto &entry : derivedStates) {
+        derivedBucketSpaceStates[entry.first] = std::make_shared<const lib::ClusterState>(entry.second);
+    }
+    return std::make_shared<const lib::ClusterStateBundle>(lib::ClusterState(baselineState), std::move(derivedBucketSpaceStates));
+}
+
+}
+
+void
+BouncerTest::abortOnlyOnDefaultBucketSpace()
```

Comment:
Consider renaming to something like `abort_request_when_derived_bucket_space_node_state_is_marked_down` to clarify that aborts are predicated on the node being down in a particular space (underscores optional, I just feel they read more clearly)